### DISCUSSION TITLE
时空幻境无法运行

### DISCUSSION THREAD

**Discussion by owl342655596** [Gamuxorg/bbs#319] [游戏或网站运行反馈]
ERROR: ld.so: object 'libgtk3-nocsd.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
./launcher.bin.x86: error while loading shared libraries: libXft.so.2: cannot open shared object file: No such file or directory

**Comment by shouhuanxiaoji** [comment]
缺依赖

**Comment by owl342655596** [comment]
> 缺依赖
该装些什么？我电脑有wine和steam

**Comment by owl342655596** [comment]
> 缺依赖

该装些什么？我电脑有wine和steam

**Comment by owl342655596** [comment]
依赖的问题我自己解决了，到gog网站上就能知道缺哪些依赖。
